autowebcompat/utils.py

```diff
@@ -251,11 +252,14 @@ def get_machine_info():
     if 'linux' not in operating_sys:
         return parameter_value_map
 
-    gpu = subprocess.check_output('lshw -C display | grep product', shell=True).strip().decode()
-    gpu = gpu.split('\n')
+    gpu = []
+    for device in device_lib.list_local_devices():
+        if device.device_type == 'GPU':
+            gpu.append(device)
     for i in range(len(gpu)):
```

Comment:
This for loop is no longer needed, you can do:
```Python
for i, device in enumerate(device_lib.list_local_devices()):
    if device.device_type != 'GPU':
        continue

    parameter_value_map['GPU_{}_name'.format(i + 1)] = device.name
    ...
```

Reply:
Done!